src/index.js

```diff
@@ -0,0 +1,90 @@
+import Store from '../modules/storeClass.js';
+
+import DisplayScreen from '../modules/display.js';
+
+import './style.css';
+
+class Task {
+  constructor(description, completed, id) {
+    this.description = description;
+    this.completed = completed;
+    this.id = id;
+  }
+}
+
+// display books event
+window.addEventListener('load', DisplayScreen.displayTasks);
+
+// Add book event
+document.querySelector('#text').addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    e.preventDefault();
+    const description = document.querySelector('#text').value;
+    const completed = false;
+    const id = Store.getTasks().length + 1;
+    // instatiate books
+    const task = new Task(description, completed, id);
+    // Display Book to screen
+    DisplayScreen.addTaskToList(task);
+    // Add to local Storage
+    Store.addTask(task);
+    // eslint-disable-next-line no-restricted-globals
```

Comment:
- Please, do not disable the linter, but rather find out how they can be solved. This will improve your project performance